contrib/packs/actions/download.yaml

```diff
@@ -5,23 +5,15 @@
   enabled: true
   entry_point: "pack_mgmt/download.py"
   parameters:
-    packs:
-      type: "array"
-      items:
-        type: "string"
-    repo_url:
+    name_or_url:
       type: "string"
-      default: "StackStorm/st2contrib"
-    branch:
+    ref:
```

Comment:
Can we call it `git_ref` or `gitref`, please? For me, it has a strong mental connection to `ref_or_id` from API layer and causes lots of confusion even after I figured what you meant by that.